cardano-api/test/Gen/Genesis.hs

```diff
@@ -0,0 +1,79 @@
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Gen.Genesis
+  ( alonzoGenesis
+  ) where
+
+import           Cardano.Prelude
+import           Hedgehog (MonadGen, Range)
+
+import qualified Cardano.Ledger.Alonzo.Genesis as Alonzo
+import qualified Cardano.Ledger.Alonzo.Language as Alonzo
+import qualified Cardano.Ledger.Alonzo.Scripts as Alonzo
+import qualified Cardano.Ledger.Coin as Ledger
+import qualified Hedgehog.Gen as Gen
+import qualified Hedgehog.Range as Range
+
+coin :: MonadGen m => Range Integer -> m Ledger.Coin
+coin r = do
+  unCoin' <- Gen.integral r
+  return $ Ledger.Coin unCoin'
```

Comment:
I think we should follow `hedgehog` style for naming generators:

* Generators should always go in a `Gen` module
* Don't use the `gen` prefix
* Always import generators `qualified as Gen`
* Always use `Gen.` to prefix at use sites.

Reply:
I don't see what's wrong with the existing style we use and I don't think it's worth it to change it all for the sake of following the `hedgehog` style.

Reply:
I've reverted to the `gen` prefix style.

You're right that it's not worth changing it for now.  I think in future we should consider that generators ought to live in the same project that declares the type and they should be packaged in a library so it can be re-used.

If we go there, then I would prefer to use the standard style.